### PR TITLE
Make the todo example works out of the box

### PR DESCRIPTION
**Pull request by poupougnac**
The todo example needs some files built with webpack. I've slightly change the main package.json to permit a global `npm run build`.

### CODE REVIEW

**Comment by poupougnac** [comment]
I have also added the possibility to add a custom HTTP server for the websocket (for people using Express).

**Comment by piglovesyou** [comment]
Thank you. Indeed it needs the way to build all sub project at once.

- The root directory should not have dependency on `webpack` that `client` already depends on.
- `runtests.sh` does the similar thing to what you wrote on `build` script. Needs refactor.
- I guess it's the best if `postinstall` does build them all so that a user doesn't need to hit a custom script, `build` .

Better idea? If no, still OK, I'll think about it after merging.

Besides that, adding `server` option to [the document](https://github.com/paldepind/synceddb/blob/master/API.md#new-serveroptions) would be appreciated.

**Comment by poupougnac** [comment]
I fixed the unnecessary dependency to webpack in the main package.json. I agree with the prepublish hook, it's a good idea. Seems okay this way ?
And of course I've also updated the doc.

File: examples/todo/README.md [hunk 1]
```diff
@@ -6,12 +6,15 @@ Try it
...
 Get the repository
 ```
-git clone https://github.com/paldepind/synceddb.git
-cd synceddb/examples/todo
+git clone https://github.com/paldepind/synceddb.git && cd synceddb/
 ```
 Get dependencies
 ```
-npm install
+npm run build
```
Comment (piglovesyou) [inline examples/todo/README.md]: Then here should be ...

**Comment by poupougnac** [comment]
Sorry I forgot to update the exemple README. Should be okay.

**Comment by piglovesyou** [comment]
It is great. Thanks again. I think I'll create another example using express based on your option.

**Comment by poupougnac** [comment]
Great idea ! Thank you for your reactivity 👍

**Pull request merged**